test/scripts/e2e_client_runner.py

```diff
@@ -66,6 +66,7 @@ def read_script_for_timeout(fname):
     return 200
 
 def _script_thread_inner(runset, scriptname):
+    time.sleep(20)
```

Comment:
I'd really like to avoid this sleep that slows down everything. Why wouldn't the system be ready yet? Haven't we already done things at this point to show that the network is up and ready? Is there a simple operation we can run in fast polling to check for things being ready?

Reply:
It seems that kmd is failing to start, so waiting 20 seconds will not help.